recipe/meta.yaml

```diff
@@ -58,3 +52,8 @@ about:
     The library supports major XML security standards.
   dev_url: https://github.com/lsh123/xmlsec
   doc_url: https://www.aleksey.com/xmlsec/documentation.html
+
+extra:
+  skip-lints:  # due to the libtool issue explained above
+    - host_section_needs_exact_pinnings
```

Comment:
You can remove this now that you pinned `libtool`

Reply:
indeed